commerce_coordinator/apps/stripe/views.py

```diff
@@ -90,10 +90,12 @@ def post(self, request):
         payment_processed_signal.send_robust(
             sender=self.__class__,
             edx_lms_user_id=payment_intent.metadata.edx_lms_user_id,
-            order_uuid=payment_intent.metadata.order_uuid,
+            order_uuid=payment_intent.metadata.order_number,
```

Comment:
While working I discovered we are trying to access the wrong attribute here. so I fixed it.

@grmartin it might be important for your testing.